packages/go_router/lib/src/matching.dart

```diff
@@ -58,6 +60,11 @@ class RouteMatchList {
   static RouteMatchList empty =
       RouteMatchList(<RouteMatch>[], Uri.parse(''), const <String, String>{});
 
+  /// Generate the full path (ex: `'/family/:fid/person/:pid'`) of a list of
+  /// [RouteMatch].
+  ///
+  /// This methods considers that [matches] is ordered accorresponding to how
```

Comment:
I can't quite parse this sentence

Reply:
I hope [docs: Better documentation](https://github.com/flutter/packages/pull/3462/commits/f12243ac360b25394893e7e0b4cbf5352ebd45be) makes it clearer